Emby.Server.Implementations/AppBase/BaseApplicationPaths.cs

```diff
@@ -114,7 +119,18 @@ public string LogDirectoryPath
         {
             get
             {
-                return Path.Combine(ProgramDataPath, "logs");
+                if (string.IsNullOrEmpty(_logDirectoryPath))
+                {
+                    _logDirectoryPath = Path.Combine(ProgramDataPath, "logs");
+
+                    Directory.CreateDirectory(_logDirectoryPath);
```

Comment:
I think you should try creating log directory always somewhere when logging starts, not only if using some default value.

Reply:
Added a check in Program.cs